src/encoded/static/components/genome_browser.js

```diff
@@ -1,610 +1,501 @@
+        && (file.file_format_type !== 'bedMethyl')
+        && (file.file_format_type !== 'bedLogR')
+        && (file.file_format_type !== 'idr_peak')
+        && (file.file_format_type !== 'tss_peak')
+        && (file.file_format_type !== 'pepMap')
+        && (file.file_format_type !== 'modPepMap')
+        && ['released', 'in progress', 'archived'].indexOf(file.status) > -1
+    ));
+    return newFileList;
+}
+
+// Fetch gene coordinate file
+export function getCoordinateData(geneLink, fetch) {
+    return fetch(geneLink, {
+        method: 'GET',
+        headers: {
+            Accept: 'application/json',
+        },
+    }).then((response) => {
+        if (response.ok) {
+            return response.json();
         }
-        files.forEach((file) => {
-            const trackLabels = this.makeTrackLabel(file);
-            if (file.file_format === 'bigWig') {
-                this.browserFiles.push({
-                    name: trackLabels.shortLabel,
-                    desc: trackLabels.longLabel,
-                    bwgURI: `${domain}${file.href}`,
-                    style: [
-                        {
-                            type: 'default',
-                            style: {
-                                glyph: 'HISTOGRAM',
-                                HEIGHT: 30,
-                                BGCOLOR: 'rgb(166,71,71)',
-                            },
-                        },
-                    ],
-                });
-            } else if (file.file_format === 'bigBed') {
-                this.browserFiles.push({
-                    name: trackLabels.shortLabel,
-                    desc: trackLabels.longLabel,
-                    bwgURI: `${domain}${file.href}`,
-                    style: [
-                        {
-                            style: {
-                                HEIGHT: 10,
-                            },
-                        },
-                    ],
+        throw new Error('not ok');
+    }).catch((e) => {
+        console.log('OBJECT LOAD ERROR: %s', e);
+    });
+}
+
+function mapGenome(inputAssembly) {
+    let genome = inputAssembly.split(' ')[0];
+    if (genome === 'hg19') {
+        genome = 'GRCh37';
+    } else if (genome === 'mm9') {
+        genome = 'GRCm37';
+    } else if (genome === 'mm10') {
+        genome = 'GRCm38';
+    }
+    return genome;
+}
+
+class GenomeBrowser extends React.Component {
+    constructor(props, context) {
+        super(props, context);
+
+        this.state = {
+            trackList: [],
+            visualizer: null,
+            showAutoSuggest: true,
+            searchTerm: '',
+            genome: '',
+            contig: props.coordinates.split(':')[0],
+            x0: +props.coordinates.split(':')[1].split('-')[0] - 5000,
+            x1: +props.coordinates.split(':')[1].split('-')[1] + 5000,
+            pinnedFiles: [],
+            disableBrowserForIE: false,
+            geneSearch: false,
```

Comment:
I may need a better solution for this. We don't want the gene search right now because the back end doesn't support it, but we want to add it back later. I don't want to remove all the code that I wrote to support autocomplete, so this is a way to turn that feature off for now.